core/clause_7_building_blocks.adoc

```diff
@@ -97,11 +97,13 @@ NOTE: This Standard only provides guidance how to represent feature data in JSON
 
 This describes the initial range of instant values. This range may be extended in the future to support additional use cases. Clients processing instant values must be prepared to receive other values. Clients may ignore values that they do not understand.
 
+NOTE: CNR Comment: What is "This"?  This clause?  This(these) example(s)?
```

Comment:
Meeting 2022-03-07: @cportele will resolve this.

Reply:
@pvretano - change the paragraph to:

"Dates and timestamps are the initial range of instant values. The range may be extended in the future to support additional use cases. Clients processing values of `time` must be prepared to receive other values. Clients may ignore values that they do not understand."

Reply:
Resolved by 2f1bd69